polyaxon_cli/run/docker.py

```diff
@@ -44,9 +45,14 @@ def _get_env_vars(project, experiment_id):
     if POLYAXON_NO_OP_KEY in os.environ:
         env_vars += [('POLYAXON_NO_OP', 'true')]
```

Comment:
I know that you did not introduce this, but is it possible to include this change in your PR:
```suggestion
        env_vars += [(POLYAXON_NO_OP_KEY, 'true')]
```

Reply:
This is just to avoid a magic string, which was not intentional. You can commit the change from the PR directly. I will merge the PR right after.